packages/gamut-templates/src/Alert/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react';
+import cx from 'classnames';
+import {
+  CardShell,
+  Container,
+  ButtonBase,
+  Button,
+  Truncate,
+} from '@codecademy/gamut';
+
+import {
+  CloseIcon,
+  ArrowChevronDownIcon,
+  ArrowChevronUpIcon,
+} from '@codecademy/gamut-icons';
+
+import { BannerType, BANNER_CONFIG } from './constants';
+import { BannerCTA } from './types';
+
+import s from './styles.module.scss';
+
+export type AlertProps = {
+  /** Banner theme string: info, alert, success, announcement, error */
+  type?: BannerType;
+  /** Toggle the display of the theme's icon */
+  showIcon?: boolean;
+  /** On close callback */
+  onClose: () => void;
+  /** Call to action configuration { text, href, onClick } */
+  cta?: BannerCTA;
+  /** Remove the max-width on the Alert container */
+  fluid?: boolean;
+  /** Number of lines to limit the message to */
+  lines?: number;
+};
+
+export const Alert: React.FC<AlertProps> = ({
+  children,
+  fluid = false,
+  type = BannerType.Info,
+  showIcon = true,
+  lines,
+  cta,
+  onClose,
+}) => {
+  const TypeIcon = BANNER_CONFIG[type];
+  const [isExpanded, setIsExpanded] = useState<boolean>(false);
+  const [isTruncated, setIsTruncated] = useState<boolean>(false);
```

Comment:
Nit, unnecessary generic:

```suggestion
  const [isExpanded, setIsExpanded] = useState(false);
  const [isTruncated, setIsTruncated] = useState(false);
```